packages/apolloschurchapp/src/content-single/UniversalContentItem/index.js

```diff
@@ -33,7 +33,7 @@ const UniversalContentItem = ({ content, loading }) => {
               </Stretchy>
             ) : null}
             <MediaControls contentId={content.id} />
-            <PaddedView vertical={false}>
+            <PaddedView vertical={coverImageSources.length || loading ? false : true}>
```

Comment:
I'm not sure there is any value in checking if we are loading or not. If we are loading we should be displaying placeholders and they would benefit from the same spacing rules.

```suggestion
            {/* fixes text/navigation spacing by adding vertical padding if we dont have an image */}
            <PaddedView vertical={!coverImageSources.length}>
```